spec/std/file_spec.cr

```diff
@@ -569,53 +608,56 @@ describe "File" do
       File.expand_path(base).should eq(base)
     end
 
-    it "converts a pathname to an absolute pathname, using ~ (home) as base" do
+    pending_win32 "converts a pathname to an absolute pathname, using ~ (home) as base" do
       File.expand_path("~/").should eq(home)
       File.expand_path("~/..badfilename").should eq(File.join(home, "..badfilename"))
-      File.expand_path("..").should eq("/#{base.split('/')[0...-1].join('/')}".gsub(%r{\A//}, "/"))
+      File.expand_path("..").should eq("/#{base.split('/')[0...-1].join('/')}".gsub(/\A\/\//, "/"))
```

Comment:
Why these changes?

Reply:
These used to be inside a `{% if flag...`, and then I found out about https://github.com/crystal-lang/crystal/issues/6180.

Not neccesary any more because they're not in a macro.